cpo-docker/bin/env_variables_all.txt

```diff
@@ -1,12 +1,16 @@
-export BEFTA_OAUTH2_CLIENT_SECRET_OF_XUIWEBAPP=OOOOOOOOOOOOOOOO
-export BEFTA_OAUTH2_ACCESS_TOKEN_TYPE_OF_XUIWEBAPP=OIDC
-export BEFTA_OAUTH2_CLIENT_ID_OF_XUIWEBAPP=xuiwebapp
-export BEFTA_OAUTH2_REDIRECT_URI_OF_XUIWEBAPP=http://localhost:3333/oauth2/callback
-export BEFTA_OAUTH2_SCOPE_VARIABLES_OF_XUIWEBAPP="profile openid roles"
-export BEFTA_S2S_CLIENT_ID_OF_XUI_WEBAPP=xui_webapp
-export BEFTA_S2S_CLIENT_SECRET_OF_XUI_WEBAPP=OOOOOOOOOOOOOOOO
+export OAUTH2_CLIENT_SECRET=OOOOOOOOOOOOOOOO
+export OAUTH2_ACCESS_TOKEN_TYPE=OIDC
+export OAUTH2_CLIENT_ID=xuiwebapp
+export OAUTH2_REDIRECT_URI=http://localhost:3333/oauth2/callback
+export OAUTH2_SCOPE_VARIABLES="profile openid roles"
+
+export BEFTA_S2S_CLIENT_ID=xui_webapp
+export BEFTA_S2S_CLIENT_SECRET=OOOOOOOOOOOOOOOO
```

Comment:
Unless/Until we have switched _CCD-Docker_ / _Data-Store_ / _Def-Store_ etc to also use `xuiwebapp` + `xui_webapp` by default in its befta configurations and environment variables then we should preserve the local config to use the indepndent variables as previouls specifeid.  **Otherwise we would be forcing developers to remeber to switch environemnt variables when they switch between projects.**

NB: All FTA test should be written to specifically state use `xuiwebapp` & `xui_webapp` sa they are in ACA rather than falling back on a default that is currently `ccd_gw`.  This process will allow jemlins to default to XUI and locally CPO to continue to run against XUI.

Reply:
Matching with Docker client config for now serves our purpose?
export OAUTH2_CLIENT_ID=xui_webapp ?

Reply:
Using XUI specific variables in local to avoid conflicts with ccd docker's defaults environment variables. @sullivangareth  you might need to add `"userTokenClientId": "xuiwebapp"` in CPO befta files[ like this ](https://github.com/hmcts/aac-manage-case-assignment/blob/e09d3cc744f2010ef96af32d69b688f9b0d266bc/src/functionalTest/resources/features/common/mca/Request_NoC__Base.td.json#L11)

Resolved in 26fa27625471be2dc96bc24962ffe3c0b12c0e03